foaf.json

```
@@ -0,0 +1,22 @@
+{
+	"@context": {
+		"foaf": "http://xmlns.com/foaf/0.1/",
+		"rdf": "http://www.w3.org/2000/01/rdf-schema#",
+	},
+	"@id": "/",
```

To my understanding, it is good practice for such a reference to resolve to an RDF document (?). In this case, the id would need to be something like `foaf.json` or `#me` instead of `/`.

---

Clients wouldn't be able to resolve `#me` as this document currently looks, right?

```suggestion
	"@id": "#",
```

---

I think they would, AFAIK using these anchors is a common practice and it should work out of the box (or am I missing something that be a problem in this case?)

---

But what would the `me` anchor point to within this document?

---

I probably should have used the term _fragments_, which can just be used to identify a sub-part of a document. This works well here, since a client that fetches the URI `https://jannisr.de/foaf.json#me` would still receive your full dataset, and it could then filter for all triples containing this exact URI as object/subject (including the fragment) to identify data for this specific entity.

Anchors in HTML/web browsers with their specific behaviour are just a special case of this more general use case, as far as I know.

---

see also: https://www.w3.org/TR/ldp-bp/#use-fragments-as-relative-identifiers